cypress/integration/page.spec.js

```diff
@@ -819,13 +876,15 @@
       });
 
       it('should not show NewCommentForm', () => {
-        newCommentForm.el().should('not.exist')
+        newCommentForm.el().should('not.exist');
```

Comment:
Similar to the previous point, ensure the form's non-existence is expected here.